client/src/components/QandA/QuestionsList.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import config from '../../../../config.js';
+import QuestionCard from './QuestionsList/QuestionCard.jsx';
+import AddQuestionOrAnswer from './QuestionsList/QuestionCard/AddQuestionOrAnswer.jsx';
+
+const host = 'https://app-hrsei-api.herokuapp.com/api/fec2/hr-rfp';
+const headers = {
+  'Authorization': `${config.TOKEN}`
+};
+
+const QuestionsList = (props) => {
+  const [questions, setQuestions] = useState([]);
+
+  useEffect(() => {
+    axios.get(host + `/qa/questions?product_id=${props.product_id}`, { headers })
+      .then(returnedQuestions => {
+        setQuestions(returnedQuestions.data.results);
+      })
+      .catch(err => console.error('Get request error (QuestionsList.jsx): ', err))
+  }, []);
+
+  return (
+    <>
+      {questions.map((item, index) =>
+        <QuestionCard
+          key={index}
```

Comment:
instead of using index for key its condired best practice to use a unique id

Reply:
Changed such that `key={item.question.id}`.